src/util/coordinate_calculation.cpp

```diff
@@ -21,10 +21,15 @@ namespace coordinate_calculation
 // Does not project the coordinates!
 std::uint64_t squaredEuclideanDistance(const Coordinate lhs, const Coordinate rhs)
 {
-    const std::uint64_t dx = static_cast<std::int32_t>(lhs.lon - rhs.lon);
-    const std::uint64_t dy = static_cast<std::int32_t>(lhs.lat - rhs.lat);
+    std::int64_t d_lon = static_cast<std::int32_t>(lhs.lon - rhs.lon);
```

Comment:
The math works out due to unsigned wraparound - so functionality wise we're fine.

The change set just makes it more explicit.